Node/Generator/.eslintignore

```diff
@@ -0,0 +1 @@
+generators/app/templates/bot.js
```

Comment:
This file file was added to the 'gitignore' list because it contains special templating syntax ('<%=') that was triggering lint errors for NodeJS.